exercises/generics/generics3.rs

```diff
@@ -0,0 +1,46 @@
+// An imaginary magical school has a new report card generation system written in rust!
+// Currently the system only supports creating report cards where the student's grade 
+// is represented numerically (e.g. 1.0 -> 5.5). However, the school also issues alphabetical grades
+// (A+ -> F-) and needs to be able to print both types of report card!
+
+// Make the necessary code changes to support alphabetical report cards, thereby making the second
+// test pass.
```

Comment:
Most of the other exercises explicitly mention the command to get the hint.

Example: https://github.com/rust-lang/rustlings/blob/master/exercises/functions/functions1.rs#L2

Not mentioning it makes it seem like there is no hint. :)

Reply:
Rustling here and I totally agree. I did the exercise a couple of minutes ago and was confused that the command for the hint missing ;)